source/clear-linux/guides/maintenance/time.rst

```diff
@@ -5,57 +5,58 @@ Set the time
 
 |CLOSIA| uses the `systemd-timesyncd.service` daemon to synchronize time.
 
-This section provides instructions to reset the time in your |CL| system when
+This guide describes how to reset the time in your |CL| system when
 the default :abbr:`NTP (Network Time Protocol)` servers cannot be reached.
 
-#. Install the `sysadmin-basic` bundle:
+#. Install the `sysadmin-basic` bundle.
 
-   .. code-block:: console
+   .. code-block:: bash
 
       swupd bundle-add sysadmin-basic
 
-#. Set your timezone. This example uses Los Angeles.
+#. Set your time zone. This example uses Los Angeles.
 
-   .. code-block:: console
+   .. code-block:: bash
 
       timedatectl set-timezone America/Los_Angeles
 
    .. note::
-      To see a list of timezones, enter:
-      `timedatectl list-timezones | grep <locale>`
+
+      To see a list of time zones, use the command:
+      :command:`timedatectl list-timezones | grep <locale>`
 
 #. Create a :file:`/etc/systemd/` directory.
 
-   .. code-block:: console
+   .. code-block:: bash
 
       mkdir -p /etc/systemd/
 
 #. Create a new file named :file:`/etc/systemd/timesyncd.conf` and enter the
-   following:
+   following text.
 
-   .. code-block:: .conf
+   .. code-block:: bash
 
       [Time]
       NTP=<Preferred Server>
       FallbackNTP=<backup server 1> <backup server 2>
 
 #. Enable the `systemd-timesyncd` service.
 
-   .. code-block:: console
+   .. code-block:: bash
 
       timedatectl set-ntp true
```

Comment:
Add "$ " to the front of the command

Reply:
Not done because it bars users from simple copy/paste.

Reply:
agree